File: vm_Assignments/vmc/L2.6/MyAsciiArg.c
#include<stdio.h>
void print_cone(int n)
{
    if(n%2 !=0)
    {
        int center = (n+1)/2;
        for(int i=1; i<center; i++)
            printf(" ");
            printf("^\n");
            for(int j=1; j<center; j++)
            {
                for(int i=1; i<center-j; i++)
                {
                    printf(" ");
                }
                for(int i=1; i<=j; i++)
                {
                    printf("%c", '/');
                }
                printf("%c", '|');
                for(int i=1; i<=j; i++)
                {
                    printf("%c", '\\');
                }
                printf("\n");
                }
            }
}
int main()
{
    print_cone(7);
}

File: vm_Assignments/vm/Beginning/L2.6/MyAsciiArg.py
def print_cone(n):
    if n % 2 != 0:
        center = ((n+1)/2)
        print(" "*int((center)) + "^")
        for i in range(1, int(center)):
            x = int(center - i)
            print(" "*x + "/"*i + "|" + "\\"*i)
# part 4 :
for i in range(10):
    print_cone(i)